tools/webdriver/webdriver/client.py

```diff
@@ -879,6 +879,10 @@ def get_computed_label(self):
     def get_computed_role(self):
         return self.send_element_command("GET", "computedrole")
 
+    @command
+    def get_accessibility_api_node(self):
```

Comment:
I'm not sure this is actually used anywhere.

Reply:
you are right!